aspnetcore/release-notes/aspnetcore-9/includes/opt_out_metrics.md

```diff
@@ -4,8 +4,8 @@
 
 HTTP requests to an endpoint can be excluded from metrics by adding metadata. Either:
 
-* Add the [`[DisableHttpMetrics]`](https://source.dot.net/#Microsoft.AspNetCore.Http.Extensions/DisableHttpMetricsAttribute.cs,258cd11ebe5f2ee1) attribute to the Web API controller, SignalR hub or gRPC service.
-* Call [DisableHttpMetrics](https://source.dot.net/#Microsoft.AspNetCore.Http.Extensions/HttpMetricsEndpointConventionBuilderExtensions.cs,7537104878c6f44a) when mapping endpoints in app startup:
+* Add the [`[DisableHttpMetrics]`](xref:Microsoft.AspNetCore.Http.DisableHttpMetricsAttribute) attribute to the Web API controller, SignalR hub or gRPC service.
```

Comment:
I'm not sure what Tom is doing, but I usually put the word "attribute" in the link when I'm showing it with brackets. In a case where I don't use brackets in the link text, I'd let the link do all of the work (`<xref:Microsoft.AspNetCore.Http.DisableHttpMetricsAttribute>`), and it would show up with the link text as `DisableHttpMetricsAttribute`. Here tho, I'd put "attribute" in the link text ...

```suggestion
* Add the [`[DisableHttpMetrics]` attribute](xref:Microsoft.AspNetCore.Http.DisableHttpMetricsAttribute) to the Web API controller, SignalR hub or gRPC service.
```